src/api/detail-page/place-detail-page.js

```
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export const fetchPlaceDetail = async (id) => {
+  try {
+    const response = await axios.get(`https://meong9.store/api/v1/places/detail/${id}`);
```

ESLint

[error] Replace ``https://meong9.store/api/v1/places/detail/${id}`` with `⏎······`https://meong9.store/api/v1/places/detail/${id}`,⏎····`

(prettier/prettier)